### PR TITLE
Nxwidget: Fix incorrect #endif position

### PR DESCRIPTION
**Pull request by halyssonJr**
## Summary
Fix incorrect #endif position
## Impact
Now users will be able to use nxwidgets with touch

## Testing
N/A

### CODE REVIEW

File: graphics/nxwidgets/src/cwindoweventhandlerlist.cxx [hunk 1]
```diff
@@ -143,8 +143,9 @@ void CWindowEventHandlerList::raiseKeyboardEvent(void)
     {
       m_eventHandlers.at(i)->handleKeyboardEvent();
     }
-#endif
 }
+#endif
+
```
Comment (acassis) [inline graphics/nxwidgets/src/cwindoweventhandlerlist.cxx:148]: Please remove extra space

**Review by acassis — APPROVED** [review]

**Comment by acassis** [comment]
@pkarashchenko any idea why our CI didn't find the extra line?

**Review by pkarashchenko — APPROVED** [review]

**Comment by pkarashchenko** [comment]
Please squash commits into one

**Review by jerpelea — APPROVED** [review]

**Review by hartmannathan — COMMENTED** [review]
The change is correct; please fix blank space and squash commits as others have said and we'll go ahead with the merge. Thanks!

**Pull request merged**